docs/legacy.md

```diff
@@ -75,15 +82,14 @@ function and is very similar. There are some important differences though:
   two effects:
 
   1. Sets the default number of sample nodes per *individual*
-  2. Changes the timescale of the coalescent process (TODO link to a section
-     that explains this effect.) By default `ploidy` is 2 and
+  2. Changes the {ref}`timescale<sec_ancestry_ploidy_coalescent_time_scales>`
+     of the coalescent process.  By default `ploidy` is 2 and
      time is scaled scaled in units of 4N generations, which is the same as
      msprime 0.x.
```

Comment:
I think this is the wrong way to say it, because time is not scaled (at least nowhere visible to the user). How about:
```
2. Changes the {ref}`time scale<sec_ancestry_ploidy_coalescent_time_scales>` over which coalescence occurs.
    By default `ploidy` is 2, and so mean time to common ancestor in a population of size `N` is `2N` generations,
    which is the same as msprime 0.x.
```

Reply:
done.